prediction/independent_poisson_model.py

```diff
@@ -117,3 +233,71 @@ def reduce_func(current_dict, pair):
 def make_prediction_output(model_outcome):
     """A constructor for a PredictionOutput of the Independent Poisson model. """
     return ConcretePredictionOutput(model_outcome)
+
+"""
+Sample result :
+iteration 10:
+1)
+    {'spain': 0.5, 
+    'brazil': 0.4, 
+    'denmark': 0.1}
+2)
+    {'colombia': 0.1,
+    'belgium': 0.1,
+    'switzerland': 0.1,
+    'france': 0.3,
+    'portugal': 0.1,
+    'spain': 0.3})
+    
+iteration 100:
+    {'brazil': 0.23,
+    'belgium': 0.16,
+    'argentina': 0.12,
+    'spain': 0.14,
+    'denmark': 0.01,
+    'croatia': 0.04,
+    'uruguay': 0.03,
+    'portugal': 0.08,
+    'colombia': 0.04,
+    'sweden': 0.02,
+    'switzerland': 0.04,
+    'mexico': 0.04,
+    'france': 0.04,
+    'england': 0.01})
+iteration 1000:
+defaultdict(int,
+            {'argentina': 0.112,
+             'portugal': 0.074,
+             'brazil': 0.217,
+             'switzerland': 0.049,
+             'croatia': 0.037,
+             'russia': 0.013,
+             'spain': 0.183,
+             'belgium': 0.066,
+             'colombia': 0.091,
+             'uruguay': 0.025,
+             'denmark': 0.025,
+             'france': 0.036,
+             'mexico': 0.026,
+             'sweden': 0.027,
+             'england': 0.018,
+             'japan': 0.001})
+iteration 10000
```

Comment:
same as above. 

The probability does not seem to add up to 1? (if these mean the probability of winning WC 2018)